trainings/sourcegraph-101/talk-track.md

```diff
@@ -0,0 +1,327 @@
+
+*Open https://docs.sourcegraph.com/integration/editor*
+
+We'll share this link with you after the presentation so you can install the right extension for you. You'll just need to point it at the right Sourcegraph instance in your settings. 
+
+### VSCode extension
+
+For VSCode users—and soon JetBrains users!—we have a more robust integration. With VSCode, you actually can do everythign we've seen so far directly in VSCode.
+
+*Share VSCode, open the Sourcegraph extension, run a search*
+
+So as you can see here, I'm able to search all of the same code that's on my Sourcegraph instance, no matter what's local. We really encourage VSCode users to install this extension, since it gives you all of Sourcegraph's search power right in the environment you're already using.
+
+### Browser extension
+
+The final thing I want to touch on is our browser extension, which you can see I have installed here.
+
+*Share browser, show the extension button, click it.*
+
+Remember the code intelligence functionality that we saw earlier? If I have this set up and pointing to my Sourcegraph instance, I have the ability to get that same functionality in my code host, which is *super* useful for code review. So, if I open a page in Sourcegraph, I can go to GitHub in our case.
+
+*Jump from whatever Sourcegraph page you were on into GitHub. Ideally, use one from our repo.*
+
+And now if I hover over any of the symbols, I get access to that same code intelligence functionality and navigation. Super helpful for code review!
+
+To install the extension, you'll want to go to this page. We'll share that in our follow-up email.
+
+*Show them https://docs.sourcegraph.com/integration/browser_extension*
+
+### Conclusion
+
+Our editors functionality lets you integrate with third-party tools, including your editor, for a seamless experience.
+
+### Resources
+
+* [VSCode extension](https://marketplace.visualstudio.com/items?itemName=sourcegraph.sourcegraph)
+* [Editor plugins](https://docs.sourcegraph.com/integration/editor)
+* [Integrations](https://docs.sourcegraph.com/integration)
+* [Browser extension](https://docs.sourcegraph.com/integration/browser_extension)
+
+## Closing thoughts
+
+Thanks so much for staying with me through this training! Today we covered our basic search functionality, our search filters, how to search commit and code history, code intelligence, notebooks, and our extensions platform. This should set you up for success in using Sourcegraph—if you have any issues, please post in Slack or shoot an email to support@sourcegraph.com so we can help you out.
+
+I want to pause here for questions—we can dive into anything you want. 
+
+*If there are questions, answer them. If not, kick it back to the AE.*
```

Comment:
```suggestion
*If there are questions, answer them.*
```

Being a public repo that we may have serve as a reference for self service type of thing or a handout to go to trainers, maybe we don't need to call out the AE steps here?

Reply:
Perhaps the disconnect here is that we want this trainings repo to be both for internal CE reference and external customer self-service before we have an LMS? If that's true, can we create sub branches here? I see value in both in lieu of an LMS. I understand this script is for CE enablement / use and not intended for a customer to discover it and use it for their own learning.

What do we think here?

Reply:
Should we make this repository private if it's only for internal use then? I thought having a public-facing repo would mean public-facing material, which is why I was suggesting removing internal language like "kick it back to the AE." Open to thoughts here for sure. I think we said in other comments maybe having a script and more of a lesson plan style, and then later some other sort of course content as well, so we'll have to see how many artifacts we're creating and what may need some rework later as we ship.

Reply:
@IAmHughes All of Sourcegraph's content is public unless there's a strong business case not to have it as such (specific customer information, some RFCs). This represents that cultural standard more than a specific external audience.

@amenne I'm happy to create a separate lesson plan document in this directory, if we want to do that. The CE could then provide that if we/as we move to a customer train-the-trainer model. I do think there's no harm in having this public; we just would have the CE give the other, more tailored content to them directly. (Even in a train-the-trainer situation, they're not going out to get this on their own, they'll be given the appropriate training and link by the CE.) That work?